### PR TITLE
feat: dex-trade driver

### PR DESCRIPTION
**Pull request by sajcics**

### CODE REVIEW

File: drivers/dextrade.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      const { base, quote } = pairs[pair];
+
+      return new Ticker({
+        base,
+        quote,
+        high: parseToFloat(ticker.high),
+        low: parseToFloat(ticker.low),
+        close: parseToFloat(ticker.last),
+        open: parseToFloat(ticker.open),
+        quoteVolume: parseToFloat(ticker.volume_24H),
```
Comment (nickpater) [inline drivers/dextrade.js:38]: If I compare the ticker (https://api.dex-trade.com/v1/public/ticker?pair=BTCUSDT) with their interface (https://dex-trade.com/spot/trading/BTCUSDT) I see that they say that is has 80k volume and in the endpoint it is around 7. So this should be the baseVolume
```suggestion
        baseVolume: parseToFloat(ticker.volume_24H),
```

**Review by nickpater — CHANGES_REQUESTED** [review]

File: drivers/dextrade.js (hunk 1 above)
Comment (sajcics) [reply drivers/dextrade.js:38]: @nickpater when I try to make a fixture again from this driver I think that their server breaks in some point of time when there is too much requests on this url `https://api.dex-trade.com/v1/public/ticker?pair=`. I constantly have `CloudflareError: 521, Web server is down`. Is it working maybe for you?

**Review by nickpater — CHANGES_REQUESTED** [review]

File: drivers/dextrade.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      const { base, quote } = pairs[ticker.pair];
+
+      return new Ticker({
+        base,
+        quote,
+        high: parseToFloat(ticker.high),
+        low: parseToFloat(ticker.low),
+        close: parseToFloat(ticker.last),
+        open: parseToFloat(ticker.open),
+        quoteVolume: parseToFloat(ticker.volume_24H), // reversed with base
```
Comment (nickpater) [inline drivers/dextrade.js:41]: If I compare the BTCUSDT ticker with their [interface](https://dex-trade.com/spot/trading/BTCUSDT) which says that they have $173K volume, I would say that it is baseVolume that they sent here.
```
{
  "id": 1041,
  "pair": "BTCUSDT",
  "last": "13036.87",
  "open": 0,
  "close": 0,
  "high": "13336.91",
  "low": "12895.27",
  "volume_24H": "13.28373978"
},
```
```suggestion
        baseVolume: parseToFloat(ticker.volume_24H),
```

**Review by nickpater — CHANGES_REQUESTED** [review]

**Review by nickpater — APPROVED** [review]

**Pull request merged**